src/baseClasses/configurationSettingsBase.ts

```diff
@@ -40,6 +44,7 @@ export default class ConfigurationSettingsBase extends Accessor {
   _handleConfigurationUpdates(e: any) {
     if (e?.data?.type === "cats-app") {
       Object.assign(this, e.data);
+      this.updateCount++;
```

Comment:
what's the count for?

Reply:
Just added as an easy way on the app side to react to a setting change occurring. It gets watched in the `assertWidgetOrdering` function